docs/index.rst

```diff
@@ -55,6 +55,16 @@ Jdaviz
 
          Jump to Mosviz
 
+   .. grid-item-card::
+      :img-top: logos/cube.svg
```

Comment:
Of course, we don't have a Rampviz logo yet, and won't until we have Jenn back. I'm leaving the cubeviz logo here in the mean time.